epochStart/metachain/economics.go

```diff
@@ -34,6 +34,7 @@ type economics struct {
 	genesisNonce          uint64
 	genesisTotalSupply    *big.Int
 	economicsDataNotified epochStart.EpochEconomicsDataProvider
+	epochEnableV2         uint32
```

Comment:
Maybe rename this to StakingV2EnableEpoch ? Enable epoch for a V2 is too general.

Reply:
done